code/backend/batch/utilities/helpers/DocumentProcessorHelper.py

```diff
@@ -42,11 +42,13 @@ def __eq__(self, other):
 
 class DocumentProcessor:
```

Comment:
Shall we rename this file whilst here and get rid of that horrible non descript word "helper"? :-D